public/video-ui/src/components/VideoAssets/VideoAssetItem.js

```diff
@@ -11,6 +11,15 @@ export default class VideoAssetItem extends React.Component {
 
   revertAsset = () => {
     this.props.revertAsset(this.props.asset.version);
+
+    // Update the activeAsset
+    let newData = Object.assign({}, this.props.video.data, {
+      activeVersion: this.props.asset.version
+    });
+
+    this.props.updateVideo(Object.assign({}, this.props.video, {
```

Comment:
this can now be `this.props.updateVideo(newData)` since there's no longer a nested structure (you could also just put the `Object.assign` inside the `updateVideo` call)